features/kernel/src/main/feature/feature.xml

```diff
@@ -20,6 +20,59 @@
     <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
     <repository>mvn:io.fabric8/fabric8-karaf-features/${fabric8.version}/xml/features</repository>
 
+    <!-- This feature replaces the jetty version provided by karaf, which has been disabled in blacklisted.properties -->
```

Comment:
✏️ Maybe add a note here and in blacklisted.properties about when these lines can be removed (i.e. when DDF is updated to a Karaf version that has a good jetty version)